fabric-rendering-v1/build.gradle

```diff
@@ -1,5 +1,5 @@
 archivesBaseName = "fabric-rendering-v1"
-version = getSubprojectVersion(project, "0.1.0")
+version = getSubprojectVersion(project, "1.0.0")
```

Comment:
0.2.0

Reply:
Hmm, most new v1 modules are 1.0.0, is anything preventing this from being marked "stable"?

Reply:
Doesnt really matter tbh

Reply:
Semver major version 0 implies possible breaking changes in any version update, so we should bump it to a stable version.